build/cert_generator/generate_kafka_certs.sh

```diff
@@ -32,7 +31,7 @@ K@fk@_store1
 K@fk@_store1
 $1
 Telco Solutions
-ACME Corp
+HPE
```

Comment:
which cert is this for ? How can the user change the same

Reply:
This script is for generating Kafka keystore certificate when not provided by the User. 
These scripts are immutable and we haven not provide any provision to make changes to script inputs, as this will require changes in docker-compose files.

Reply:
Then we have to make this configurable. Also pls use private repo to ake changes before merging it this project.

Reply:
Can we take this design changes in next Kubernetes based deployment ? Since this was designed in Phase 1 and will go away once Kubernetes based deployment is merged in upstream. 
Yes this is forked repo branch (rahulroshan-kachchap:kafka_integrate) .